articles/azure/azs-how-create-vm-cli.md

```diff
@@ -0,0 +1,96 @@
+---
+title: How to create a Virtual Machine - CLI | UKCloud Ltd
+description: Provides help for creating a virtual machine on UKCloud for Microsoft Azure using Azure CLI
+services: azure-stack
+author: Bailey Lawson
+toc_rootlink: Users
+toc_sub1: How To
+toc_sub2: Create a Virtual Machine
+toc_sub3:
+toc_sub4:
+toc_title: Create a Virtual Machine - CLI
+toc_fullpath: Users/How To/Create a Virtual Machine/azs-how-create-vm-cli.md
+toc_mdlink: azs-how-create-vm-cli.md
+---
+
+# How to create a Virtual Machine in Azure Stack with Azure CLI
+
+## Overview
+
+With UKCloud for Microsoft Azure, you can leverage the power of Microsoft Azure to create virtual machines (VMs) for your on-premises applications. 
+As UKCloud for Microsoft Azure is built on UKCloud’s assured, UK-sovereign multi-cloud platform, those applications can work alongside other cloud platforms, such as Oracle,
+VMware and OpenStack, and benefit from native connectivity to non-cloud workloads in Crown Hosting and government community networks, including PSN, HSCN and RLI.
+
+## Prerequisites
+
+Ensure your Azure CLI environment is setup as detailed in [Configure the Azure Stack user's PowerShell environment](azs-how-configure-cli.md).
+
+## Creating a resource group
+
+Before creating a virtual machine, it is necessary to create a resource group to deploy the VM to. Note that this step can be skipped if you already have created the resource group to deploy the VM to.
+
+```Bash
+az group create --name '<Resource Group Name>' --location 'frn00006'
+```
+
+## Creating a virtual machine
+
+Once the Resource group has been created, you must choose which image template to use for your VM. To obtain a list of available images run the following command:

```

Comment:
When you've created the resource group, you must choose which image template to use for your VM.